src/api/server/common.rst

```diff
@@ -938,6 +939,22 @@ error.
     :code 200: Request completed successfully
     :code 401: CouchDB Server Administrator privileges required
 
+    The ``info`` field of a scheduler doc:
```

Comment:
I tried a few things but couldn't get it to work either

Reply:
I'm officially on break for a week from CouchDB, but this might help: http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#hyperlink-targets